packages/ip2location/ip2location.8.0.0/opam

```diff
@@ -0,0 +1,46 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "IP2Location OCaml module to get geolocation data"
+description:
+  "IP2Location IP Geolocation OCaml module enables you to get your visitors' country, region, city, latitude, longitude, ZIP code, ISP, domain name, timezone, connection speed, IDD code, area code, weather station code, weather station name, MCC, MNC, mobile brand name, elevation, usage type, address type and IAB category based on IP address."
+maintainer: ["IP2Location.com"]
+authors: ["IP2Location.com"]
+license: "MIT"
+tags: ["ip2location" "geolocation"]
+homepage: "https://github.com/ip2location/ip2location-ocaml"
+doc: "https://github.com/ip2location/ip2location-ocaml"
+bug-reports: "https://github.com/ip2location/ip2location-ocaml/issues"
+depends: [
+  "ocaml"
+  "dune" {>= "3.4"}
+  "stdint"
+  "ipaddr"
+  "lwt"
+  "cohttp"
+  "cohttp-lwt-unix"
+  "yojson"
```

Comment:
You're using `Yojson.Safe.t`
```suggestion
  "yojson" {>= "1.6.0"}
```

Reply:
Thanks, your suggestions fixed those errors. Unfortunately,  still having failures for arm32 and x86_32. Is it because I'm using Uint128?